### PR TITLE
fix(code-snippet): Update bottom spacing & fix bug to only show more btn when needed

### LINKED ISSUE #949: Code snippet bug
Code in the code snippet is overflowing onto the ghost button. Was expecting it to push the code to a new line when it got within 16px of the ghost button (see image below

Current:
screen shot 2018-06-28 at 7 06 54 pm

Proposed:
image

### PR DESCRIPTION
**Pull request by alisonjoseph**
## Overview

Resolves #949 

Adds an additional 1rem of padding to the bottom of the multiline code snippet to make sure the show more button doesn't overlap text. 

screen shot 2018-07-12 at 11 51 15 am
screen shot 2018-07-12 at 11 50 42 am
screen shot 2018-07-12 at 11 50 37 am

@aagonzales can you take a look and make sure the spacing is ok here.

### CODE REVIEW

File: src/components/code-snippet/code-snippet.hbs [hunk 1]
```diff
@@ -11,29 +11,11 @@
...
-    modal : 9000,
-    overlay : 8000,
-    dropdown : 7000,
-    header : 6000,
-    footer : 5000,
-    hidden : - 1,
-    overflowHidden: - 1,
-    floating: 10000
-  );
-      </code>
```
Comment (tw15egan) [inline src/components/code-snippet/code-snippet.hbs]: Do we want to remove all this?

Comment (alisonjoseph) [reply src/components/code-snippet/code-snippet.hbs]: Oops. Nope. That was just me testing

**Review by tw15egan — APPROVED** [review]
Looks good on the code side 👍

**Review by aagonzales — APPROVED** [review]
Looks like it solve the overlap problem, so its good to go! 👍 ✅

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @alisonjoseph!

**Pull request merged**